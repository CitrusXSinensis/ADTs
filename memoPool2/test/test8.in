ra0x
ca
pa
fa 1
fa 0
