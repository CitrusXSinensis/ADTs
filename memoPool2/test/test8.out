1/4
0
Nullptr
Nullptr
